internal/backendstore/backendstore.go

```diff
@@ -66,6 +66,26 @@ func (b *BackendStore) GetBackendClients(beNames []string) map[string]S3Client {
 	return clients
 }
 
+func (b *BackendStore) GetFirstActiveBackendClient(beNames []string) S3Client {
```

Comment:
For me this function is a bit overkill, while i understand the performance reason. My version would be:

```
func (b s3Backends) getFirst() S3Client {
	for _, v := range b {
		return v.s3Client
	}

	return nil
}
```

```
GetActiveBackends().First()
```